### PR TITLE
Decide `LiveBlogEpic` position based on if there is a permalink

### PR DESCRIPTION
**Pull request by oliverlloyd**
## What does this change?
This PR moves the decision about where to position the `LiveBlogEpic` from the server to the client. We also now animate in the epic when a permalink is in use to reduce any confusion from new content suddenly appearing on the page

## Why?
This allows us to take into consideration if a permalink is being used and choose to place the epic below the linked to block in that case. This prevents CLS from breaking the permalink experience

This PR replaces https://github.com/guardian/dotcom-rendering/pull/4629. After suggestions from @tomrf1 @TomPretty I was able to implement a solution that fixed the CLS problem with permalinks without reducing the volume of impressions for the epic. Indeed, this could potentially increase conversions as we're now making the epic more prominent for those 8% of readers using permalinks

| Before      | After      |
|-------------|------------|
| before | after |

### CODE REVIEW

**Review by tomrf1 — APPROVED** [review]
Looks good - such a nice solution!

**Comment by TomPretty** [comment]
Yeah looks great to me - glad to see it wasn't too much extra effort and I think this makes for a neat solution!

File: dotcom-rendering/src/web/components/LiveBlogEpic.importable.tsx [hunk 1]
```diff
@@ -201,6 +202,12 @@ const Render = ({
 	);
 };
 
+/**
+ * Using the payload, make a fetch request to contributions service
+ *
+ * @param payload - unknown
+ * @param contributionsServiceUrl - string
```
Comment (mxdvl) [inline dotcom-rendering/src/web/components/LiveBlogEpic.importable.tsx:209]:
```suggestion
 * @param {unknown} payload
 * @param {string} contributionsServiceUrl
```

**Review by mxdvl — COMMENTED** [review]
I'm not entirely sure the animation works, but otherwise this is a great approach!!

**Pull request merged**